errors.go

```diff
@@ -0,0 +1,29 @@
+package main
```

Comment:
Created this in order to stop repeating the changes to the response template.

Can probably do something similar with other logic in `main.go`